### DISCUSSION TITLE
How to trigger a DAG from AWS on an old Airflow instance?

### DISCUSSION THREAD

**Discussion by binarykitchen** [apache/airflow#37958] [Q&A]
Hello all,

In my company we unfortunately run an ancient Airflow instance v1.8.0 we cannot upgrade right now.

There is one problem we're trying to crack: how to run a DAG by a chosen AWS SQS signal?

The documentation for that old version has disappeared. Any ideas or even hacks are very welcome. Thank you :)

Cheers, Michael

**Comment by potiuk** [comment]
I think it's your only chance to follow that guy:

![image](https://github.com/apache/airflow/assets/595491/2f08d0cc-147c-4726-805f-d4f09cd66f5a)

It's never that you "CAN'T" upgrade. It's always the matter of cost of the upgrade. You have to think (and especially your company management has to make the decision). If it is more costly to pay your salary to try to find some ways around things like that or to upgrade.

Because there is literally 0 chance you will get someone spending more time on your issue than me writing this message, so you are generally on your own with your question.

**Reply by binarykitchen** [reply]
@potiuk Appreciate your honesty, you can tell me that. Basically, agree with you.

Just checking all options.
